### PR TITLE
[MISP Feed] Update test-module

### PR DESCRIPTION
**Pull request by MichaelYochpaz**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
relates: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-23017)

## Description
Replace the test-module of MISP Feed to an endpoint that returns a much smaller response, to avoid timeouts while using the test-module due to large responses.

### CODE REVIEW

File: Packs/FeedMISP/ReleaseNotes/1_1_0.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### MISP Feed
+
+- Update test button functionality to use a different endpoint to test connection as the previously used one could cause timeouts.
```
Comment (ShirleyDenkberg) [inline Packs/FeedMISP/ReleaseNotes/1_1_0.md:6]:
```suggestion
- Updated the test button functionality to use a different endpoint to test the connection as the previously used endpoint could cause timeouts.
```

File: Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.yml [hunk 2]
```diff
@@ -82,29 +79,23 @@ configuration:
...
   type: 8
   required: false
-  additionalinfo: When selected, the exclusion list is ignored for indicators from
-    this feed. This means that if an indicator from this feed is on the exclusion
-    list, the indicator might still be added to the system.
+  additionalinfo: When selected, the exclusion list is ignored for indicators from this feed. This means that if an indicator from this feed is on the exclusion list, the indicator might still be added to the system.
 - display: MISP Attribute Tags
-  additionalinfo: Attributes having one of the tags, or being an attribute of an event having one of the tags, will be returned. You can enter a comma-separated list of tags,
-    for example <tag1>,<tag2>,<tag3>. The list of MISP tags can be found in your MISP instance under 'Event Actions'>'List Tags'
+  additionalinfo: Attributes having one of the tags, or being an attribute of an event having one of the tags, will be returned. You can enter a comma-separated list of tags, for example <tag1>,<tag2>,<tag3>. The list of MISP tags can be found in your MISP instance under 'Event Actions'>'List Tags'
```
Comment (ShirleyDenkberg) [inline Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.yml:84]:
```suggestion
  additionalinfo: Attributes having one of the tags, or being an attribute of an event having one of the tags, will be returned. You can enter a comma-separated list of tags, for example <tag1>,<tag2>,<tag3>. The list of MISP tags can be found in your MISP instance under 'Event Actions'>'List Tags'.
```

File: Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.yml [hunk 3]
```diff
@@ -82,29 +79,23 @@ configuration:
...
   name: attribute_tags
   required: false
   type: 0
 - display: MISP Attribute Types
-  additionalinfo: Attributes of one of these types will be returned. You can enter a comma-separated list of types,
-    for example <type1>,<type2>,<type3>. The list of MISP types can be found in your MISP instance then 'Event Actions'>'Search Attributes'>'Type dropdown list'
+  additionalinfo: Attributes of one of these types will be returned. You can enter a comma-separated list of types, for example <type1>,<type2>,<type3>. The list of MISP types can be found in your MISP instance then 'Event Actions'>'Search Attributes'>'Type dropdown list'
```
Comment (ShirleyDenkberg) [inline Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.yml:89]:
```suggestion
  additionalinfo: Attributes of one of these types will be returned. You can enter a comma-separated list of types, for example <type1>,<type2>,<type3>. The list of MISP types can be found in your MISP instance then 'Event Actions'>'Search Attributes'>'Type dropdown list'.
```

**Comment by ShirleyDenkberg** [comment]
@bziser Doc review completed

File: Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py [hunk 4]
```diff
@@ -504,23 +497,17 @@ def update_indicator_fields(indicator_obj: Dict[str, Any], tlp_color: Optional[s
 """
 
 
-def test_module(client: Client, params: Dict[str, str]) -> str:
-    """Builds the iterator to check that the feed is accessible.
+def test_module(client: Client) -> str:
+    """
+    Test connectivity to server.
```
Comment (MichaelYochpaz) [inline Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py:502]:
```suggestion
    Fetch a single feed item to assure configuration is valid.
```

**Review by bziser — APPROVED** [review]

**Pull request merged**